recipes/vscode-jupyter/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "vscode-jupyter" %}
+{% set version = "2020.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/microsoft/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 99502db43f9f2ac4185d55ae51a70c81ffe9353983dcaca5cd1a7cf29f783a11
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports:
+    # node-canvas is built but not installed, so ignore its dependencies
+    - cairo
+    - giflib
+    - libglib
+    - libjpeg-turbo
+    - pango
+    - pixman
+  os_version:
+    linux_64: cos7
```

As I told you in private message: This needs to go into `conda_forge.yaml`  in the feedstock, please just skip linux here and we can activate it later once a feedstock has been created.

```suggestion
  os_version:
    linux_64: cos7
```

---

Removed again. Also skipping the linux build for now.